src/component/ResponsiveContainer.tsx

```diff
@@ -54,14 +54,19 @@ export const ResponsiveContainer = forwardRef(
       minHeight,
       maxHeight,
       children,
-      debounce = 0,
+      // debounce = 0,
```

Comment:
why comment this out?

Reply:
I haven't implemented it yet, so I've commented it out